payload/src/main/java/io/jenkins/jenkinsfile/runner/FileSystemSCM.java

```diff
@@ -57,4 +66,23 @@ public SCMRevisionState calcRevisionsFromBuild(@Nonnull Run<?, ?> build, @Nullab
     public PollingResult compareRemoteRevisionWith(@Nonnull Job<?, ?> project, @Nullable Launcher launcher, @Nullable FilePath workspace, @Nonnull TaskListener listener, @Nonnull SCMRevisionState baseline) throws IOException, InterruptedException {
         return PollingResult.NO_CHANGES;
     }
+
+    public DescriptorImpl getDescriptor() {
+        return (DescriptorImpl) Jenkins.getInstance().getDescriptorOrDie(FileSystemSCM.class);
```

Comment:
@jglick this is throwing an AssertionError because it does not seem to find the extension. Would you have any clue?

Reply:
You should not need to add this override. And no I do not know why the `@Extension` would not be working.